packages/widget/src/controllers/transfers/fungible-token-transfer.ts

```diff
@@ -45,6 +45,7 @@ export class FungibleTokenTransferController implements ReactiveController {
   public supportedSourceNetworks: Domain[] = [];
   public supportedDestinationNetworks: Domain[] = [];
   public supportedResources: Resource[] = [];
+  public displayAmountOnDestination: boolean = false;
```

Comment:
Is this really needed? Wouldn't check ´pendingEvmTransferTransaction != undefined˙ do the same?

Reply:
let me try, my idea is to avoid rendering the value before it's finalize the computation

Reply:
I know but we have pnding tx only after we finish computation

Reply:
solved